build-ps/percona-server-5.7_builder.sh

```diff
@@ -716,14 +716,25 @@ build_tarball(){
     export CXXFLAGS="${CFLAGS}"
     if [ "${YASSL}" = 0 ]; then
         if [ -f /etc/redhat-release ]; then
-            SSL_VER_TMP=$(yum list installed|grep -i openssl|head -n1|awk '{print $2}'|awk -F "-" '{print $1}'|sed 's/\.//g'|sed 's/[a-z]$//')
-            export SSL_VER=".ssl${SSL_VER_TMP}"
+            if [ "${RHEL}" -eq 8 ]; then
+                SSL_VER_TMP=$(yum list installed|grep -i openssl|head -n1|awk '{print $2}'|awk -F ":" '{print $2}'|awk -F "-" '{print $1}'|sed 's/\.//g'|sed 's/[a-z]$//')
+            else
+                SSL_VER_TMP=$(yum list installed|grep -i openssl|head -n1|awk '{print $2}'|awk -F "-" '{print $1}'|sed 's/\.//g'|sed 's/[a-z]$//')
+            fi
         else
-            SSL_VER_TMP=$(dpkg -l|grep -i libssl|grep -v "libssl\-"|head -n1|awk '{print $2}'|awk -F ":" '{print $1}'|sed 's/libssl/ssl/g'|sed 's/\.//g')
-            export SSL_VER=".${SSL_VER_TMP}"
+            SSL_VER_TMP=$(dpkg -l|grep -i libssl|head -n1|awk '{print $3}'|awk -F "." '{print $1 $2}')
```

Comment:
```
dpkg -l|grep -i libssl|head -n1|awk '{print $3}'
1.0.2n-1ubuntu5.3
```
and your command returns:
```
dpkg -l|grep -i libssl|head -n1|awk '{print $3}'|awk -F "." '{print $1 $2}'
10
```
previously it was ssl100 which is also incorrect as version was

```
dpkg -l|grep -i libssl|grep -v "libssl\-"| dpkg -l|grep -i libssl|grep -v "libssl\-"
ii  libssl1.0.0:amd64                               1.0.2n-1ubuntu5.3                                           amd64        Secure Sockets Layer toolkit - shared libraries
ii  libssl1.1:amd64                                 1.1.1-1ubuntu2.1~18.04.5                                    amd64        Secure Sockets Layer toolkit - shared libraries
```